css/evaluation.css

```diff
@@ -0,0 +1,76 @@
+body {
+  position: relative;
+  margin: 0;
+  padding-bottom: 6rem;
+  min-height: 100%;
+}
+
+
+#tbHeader {
```

Comment:
Would we want to stick to a certain naming convention (e.g., `camelCase`, `spinal-case`, `snake_case`) across the entire app? Would something like that make sense?